portfolio2.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0,user-scalable=no">
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+<meta name="HandheldFriendly" content="true">
+	<title>Simona Ambrosini-Portfolio</title>
+	<link rel="stylesheet" href="portfolio2.css">
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,600;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+	<nav class="navbar">
+		<div class="title">
+			<a href="#" class="toggle-button">
+				<span class="bar"></span>
+				<span class="bar"></span>
+				<span class="bar"></span>
+		</div>
+		<div class="navbar-link">
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#about">About Me</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#contact">Contact Me</a></li>
+		</div>
+	    </div>
+			
+	</nav>
+	
+	<section id="home">
+		<div class="central">
+			<h1 class="headings">Hello!<br> I'm Simona Ambrosini.</h1>
+			 <button class="btn" href="#contact">Hire Me</button>
+		 </div>
+	 </section>
+			 
+			 
+			 <section id="about">
+				 <div class="about">
+				   <h1>About Me</h1>
+			     </div>
+				 <div id="pic">
+					 <img src="icon.png" alt="woman">
+				 </div>
+					 <div id="intro">
+						 <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Sed ut perspiciatis, unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam eaque ipsa, quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt, explicabo.</p>
+					 </div>
+			 </section>
+			 
+			 <section id="portfolio">
+			 	<h1 class="headings">Portfolio</h1>
+				<div class="gallery">
+					<div class="project">
+						<h2>Project 1</h2>
+						<div class="square">
+						<p>First Project</p>
+					    </div>
+					<img src="1.jpg">
+				    </div>
+					<div class="project">
+						<h2>Project 2</h2>
+						<div class="square">
+						<p>Second Project</p>
+					    </div>
+						<img src="1.jpg">
+					</div>
+					<div class="project">
+						<h2>Project 3</h2>
+						<div class="square">
+						<p>Third Project</p>
+						</div>
```

Comment:
'double div-ing' here, a `<p>` tag doesn't need to be put in a container on it's own. The general rule of thumb is that if you want to group multiple elements together, you may want a `<div>`, otherwise an element can be placed in the HTML on its own